setup-saturn.sh

```
@@ -0,0 +1,57 @@
+# Set for current session
+export OPENROUTER_API_KEY="$api_key"
+
+# Optional: Install as global tool
+echo "Would you like to install Saturn as a global .NET tool?"
+echo "This will allow you to run 'saturn' from anywhere."
+read -p "Install globally? (y/n): " install_global
+
+if [ "$install_global" = "y" ] || [ "$install_global" = "Y" ]; then
+    echo ""
+    echo "📦 Installing Saturn as global tool..."
+    cd /Users/jlee/projects/_tools_/Saturn
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)